tests/init/pos/always-valid.scala

```diff
@@ -0,0 +1,13 @@
+final class MyNeClass(o: MyNeClass) {
```

Comment:
I'd suggest renaming to a filename that includes eq and ne, e.g. `eq-ne-always-valid.scala`.